config/ext_jenkins-release_privatek8s.yaml

```
@@ -0,0 +1,692 @@
+      #             securityGroups: "ec2_agents_release"
+      #             stopOnTerminate: false
+      #             t2Unlimited: false
+      #             tags:
+      #             - name: "architecture"
+      #               value: "arm64"
+      #             - name: "os"
+      #               value: "linux"
+      #             - name: "jenkins"
+      #               value: "release.ci.jenkins.io"
+      #             type: T4gMedium
+      #             useEphemeralDevices: true
+      #           - ami: "ami-0c3f1d709a2270b04" # https://github.com/jenkins-infra/packer-images/
+      #             amiOwners: "200564066411"
+      #             amiType:
+      #               unixData:
+      #                 sshPort: "22"
+      #                 # Windows VM agent are not supported yet: https://issues.jenkins.io/browse/JENKINS-69304
+      #                 # slaveCommandPrefix: "PATH=C:/tools/jdk-11/bin:^${PATH}"
+      #             associatePublicIp: true
+      #             connectBySSHProcess: false
+      #             connectionStrategy: PUBLIC_IP
+      #             deleteRootOnTermination: true
+      #             description: "Windows 2019 AMD64"
+      #             ebsOptimized: false
+      #             hostKeyVerificationStrategy: ACCEPT_NEW     # TODO: set it CHECK_NEW_HARD (slower but safer)
+      #             idleTerminationMinutes: "30"                # Windows instances are billed per hour, let's reuse
+      #             instanceCapStr: "10"
+      #             labelString: "windows windows-amd64 windows-amd64-docker docker-windows"
+      #             launchTimeoutStr: "600"                     # Wait for Windows to start all the EC2 launch services
+      #             maxTotalUses: 10                            # Windows instances are billed per hour, let's reuse
+      #             minimumNumberOfInstances: 0
+      #             minimumNumberOfSpareInstances: 0
+      #             mode: EXCLUSIVE
+      #             monitoring: false
+      #             numExecutors: 1
+      #             remoteAdmin: "Administrator"
+      #             remoteFS: "C:\\Jenkins"
+      #             securityGroups: "ec2_agents_release"
+      #             stopOnTerminate: false
+      #             t2Unlimited: false
+      #             tags:
+      #             - name: "architecture"
+      #               value: "amd64"
+      #             - name: "os"
+      #               value: "windows"
+      #             - name: "jenkins"
+      #               value: "release.ci.jenkins.io"
+      #             tenancy: Default
+      #             tmpDir: "C:\\\\Temp"                          # Mandatory because EC2 plugins assumes SSH = Unix and searches for /tmp
+      #             type: T3Medium
+      #             useEphemeralDevices: true
+      jobs-settings: |
+        jobs:
+          - script: >
+              folder('components') {
+                displayName('Components (on privatek8s)')
```

Temporary marker to be sure of the instance, to be removed:
```suggestion
                displayName('Components')
```